browser/src/map/handler/Map.SlideShow.js

```diff
@@ -147,16 +147,27 @@ L.Map.SlideShow = L.Handler.extend({
 	_startPlaying: function() {
 		// Windowed Presentation
 		if (this._presentInWindow) {
-			this._slideShowWindowProxy = window.open(this._slideURL, '_blank', 'popup'); // do we need to set this to null when closed or is that already done?
+
+			var popupTitle = "Windowed Presentation: " + this._map['wopi'].BaseFileName;
```

Comment:
This has to be translated string. Please use `_('....')`.
Also everywhere we used `'` for strings here. Can we be consistent inside single file? (somehow we have enabled `"` in master now)